src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/devices-page.mdx

```diff
@@ -1,65 +1,67 @@
-  alt="screen mobile devices.png"
+  title="Mobile Devices page"
+  alt="Screenshot of Devices page"
   src={mobileDevicesSummary}
 />
 
-<figcaption>
-  **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > App > Devices:** Use this page to view, sort, or drill down into detailed information about the top five types of devices using your mobile app, including interaction and HTTP request times, error rates, and active users.
-</figcaption>
 
-To view performance details about your users' mobile devices:
+## View the Devices page [#view-devices-page]
 
-1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > App > Devices**.
-2. To select the mobile app versions or change the time period, use the **Versions** menu and [**time picker**](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#dash-time-picker) below the New Relic menu bar.
-3. Optional: Select the **Sort by** and **Hide &lt; 1% throughput** options.
-4. To limit details to a specific device **type** (for example, iPad), select its name.
+1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile.
+2. Select your mobile app.
+3. In the left-hand menu, click **Devices**.
 
-## Viewing drill-down details [#details]
+
+## Drill down into specific metrics [#drill-down]
 
 To drill down into detailed information, use any of New Relic's standard [user interface functions](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform). In addition:
 
 * To view a list of specific devices or models (for example, iPad mini, iPad Air, etc.), select the **type** (for example, iPad).
 * To view details for a specific device or model, select its name from the expanded list.
 * To view trace details a slow transaction (if available), select its link. For more information, see [Interactions page](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/interactions-page).
-* To return to the main Devices page, select the **Close** (**X**) button.
 
 <img
-  title="screen mobile devices drilldown.png"
-  alt="screen mobile devices drilldown.png"
+  title="Drill down into device metrics"
+  alt="Gif of specific device metrics"
   src={mobileDeviceDrilldown}
 />
 
-<figcaption>
-  **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > App > Devices > (select a device)**: This page provides drill-down details for the selected device, including http response time, network failures, active sessions, and slowest transaction traces (if available).
-</figcaption>
+## Create alerts [#create-alerts]
+
+Now that you've got visibility into your app's performance, it's time to set up alerts for key performance changes. 
+
+Simply click into a metric, then from the chart menu, click **Create alert condition** and follow the steps in the UI.
```

Comment:
"Click into" reads a little awkwardly, maybe just "select a metric". I'm also not sure you need the "Simply" at all.